### PR TITLE
enh(ui): replace icomoon by mui icons in Extension manager

### PR DESCRIPTION
**Pull request by bdauria**
## Description

Capture d’écran 2021-09-14 à 15 49 28

Capture d’écran 2021-09-14 à 15 49 42

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Play with the extension manager

### CODE REVIEW

File: www/front_src/src/Extensions/ExtensionsHolder/index.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+                        {licenseInfo.label}
+                      </Typography>
+                    </Paper>
+                  )}
+                </Card>
+              </Grid>
+            );
+          })}
+        </Grid>
+        {/* </Paper> */}
```
Comment (Thebarda) [inline www/front_src/src/Extensions/ExtensionsHolder/index.js:213]: Remove this comment

File: www/front_src/src/Extensions/ExtensionsHolder/index.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+              >
+                <Card
+                  style={{ display: 'grid', height: '100%' }}
+                  variant="outlined"
+                >
+                  <div style={{ height: 10 }}>
+                    {isLoading && <LinearProgress />}
+                  </div>
+                  <CardContent>
+                    {/* {entity.version.installed ? <InfoIcon /> : null} */}
```
Comment (Thebarda) [inline www/front_src/src/Extensions/ExtensionsHolder/index.js:123]: Remove this comment

File: www/front_src/src/Extensions/ExtensionsHolder/index.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  getPropsFromLicense = (licenseInfo) => {
+    if (licenseInfo && licenseInfo.required) {
+      if (!licenseInfo.expiration_date) {
+        return {
+          color: '#f90026',
+          label: 'License required',
+        };
+      }
+      if (!isNaN(Date.parse(licenseInfo.expiration_date))) {
+        // @todo use moment to convert date in the proper format (locale and timezone from user)
```
Comment (Thebarda) [inline www/front_src/src/Extensions/ExtensionsHolder/index.js:43]: Remove this comment

File: www/front_src/src/Extensions/ExtensionDetailsPopup/index.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+          <Grid item>
+            <Divider />
+          </Grid>
+          <Grid item>
+            {loading ? (
+              <ReleaseNoteSkeleton animate={animate} />
+            ) : (
+              <Link href={modalDetails.release_note}>
+                {modalDetails.release_note}
+              </Link>
```
Comment (Thebarda) [inline www/front_src/src/Extensions/ExtensionDetailsPopup/index.js:159]: Wrap it within a Typography

**Review by Thebarda — COMMENTED** [review]
Where and how the images (like: `www/front_src/src/Extensions/img/content-horizontal-line-blue.png`) are used ?

File: www/front_src/src/Extensions/ExtensionsHolder/index.js (hunk 2 above)
Comment (bdauria) [reply www/front_src/src/Extensions/ExtensionsHolder/index.js:123]:
```suggestion
```

File: www/front_src/src/Extensions/ExtensionsHolder/index.js (hunk 1 above)
Comment (bdauria) [reply www/front_src/src/Extensions/ExtensionsHolder/index.js:213]:
```suggestion
```

File: www/front_src/src/Extensions/ExtensionsHolder/index.js (hunk 3 above)
Comment (bdauria) [reply www/front_src/src/Extensions/ExtensionsHolder/index.js:43]:
```suggestion
```

**Review by Thebarda — DISMISSED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]

**Pull request merged**